### PR TITLE
Added Admin page and filled out data for books. The view pages were also modified.

### PR DESCRIPTION
**Pull request by RizwanNiazFolio3**
### Current Status
Currently the site has no actual records and there is no way to access the admin page.

### Changes/Additions
The admin.py file in catalog/ was changed to add content for the classes currently defined in models.py

After logging in the admin can see the site and add data
Admin page

Book Instances

Authors

### Feedback
I'd like general feedback on the code and pull request itself, any changes I should add to the code to make sure it follows the style guide. Best practices etc. 

Also, the challenge questions at the end of the tutorial asked us to add the inline list of books made by an author as shown in the last picture above, but the list content was so big that some of the data is too far to the side

authors in line

This will probably addressed further on in another tutorial but i'm curious if there's a simple fix for this that i'm just not seeing right now

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.pyc
```
Comment (shahabejaz) [inline .gitignore:1]: New line!

File: catalog/admin.py [hunk 2]
```diff
@@ -1,3 +1,49 @@
 from django.contrib import admin
 
+from .models import Author, Genre, Book, BookInstance, Language
+
 # Register your models here.
```
Comment (shahabejaz) [inline catalog/admin.py:5]: You can remove this.

**Review by shahabejaz — COMMENTED** [review]
Two minor comments, otherwise looks good.

File: .gitignore (hunk 1 above)
Comment (RizwanNiazFolio3) [reply .gitignore]: Done.

File: catalog/admin.py (hunk 2 above)
Comment (RizwanNiazFolio3) [reply catalog/admin.py:5]: I don't think I understand. I commented out line 3 to see what happens and the website wouldn't load.

Comment (shahabejaz) [reply catalog/admin.py:5]: Oh not line 3.. line 5 but its not very important.

**Review by shahabejaz — APPROVED** [review]

**Pull request merged**